### PR TITLE
new package: davegnukem

### LINKED ISSUE #352: [Package Help]: Dave Gnukem
### Name

Dave Gnukem

### Home page URL

https://djoffe.com/gnukem/

### Source code URL

https://github.com/davidjoffe/dave_gnukem

### Packaging policy acknowledgement

- [X] I understand [Tur Package Policy](https://github.com/termux-user-repository/tur/wiki/TUR-package-policy.).

### Additional information

Well - I wanted to move the Dave Gnukem application to Termux. On my phone compiled by hand it works very well. So I started preparing a script that would compile it in the build environment on Termux. The code also compiled correctly, but it doesn't find the libSDL2 library, which is on the system. I have no idea where this problem is coming from.

Of course if anyone has ideas on how to improve the build process, I'm always keen to hear good advice as I'm no expert.

build.sh:
https://paste.hostux.net/?3bc6f627818f25a8#48SFnEBxY7pvh7ysfMp4JPPCMvNPmzyYzboAWuZ5VoK1

```
~ davegnukem
CANNOT LINK EXECUTABLE "davegnukem": library "libSDL2-2.0.so" not found: needed by main executable
```

### PR DESCRIPTION
**Pull request by licy183**
Closes #352.

### CODE REVIEW

**Comment by IntinteDAO** [comment]
@licy183 

One proposal. 
https://github.com/termux-user-repository/tur/pull/355/commits/1c55a30068c034e1f3992750823141f4907abd40#diff-7a37231957f9339dcc9a65549560cc3897bc990fb9c097fe15215b15ceb3cd0bR19

I also suggest adding the -f parameter, because if you build the game 2 times, the link stays and during the second build, the build script detects an error and does not generate a new package.

so in short

```
ln -fsr $TERMUX_PREFIX/games/davegnukem $TERMUX_PREFIX/bin/davegnukem
```

**Comment by licy183** [comment]
> @licy183
> 
> One proposal. [1c55a30#diff-7a37231957f9339dcc9a65549560cc3897bc990fb9c097fe15215b15ceb3cd0bR19](https://github.com/termux-user-repository/tur/commit/1c55a30068c034e1f3992750823141f4907abd40#diff-7a37231957f9339dcc9a65549560cc3897bc990fb9c097fe15215b15ceb3cd0bR19)
> 
> I also suggest adding the -f parameter, because if you build the game 2 times, the link stays and during the second build, the build script detects an error and does not generate a new package.
> 
> so in short
> 
> ```
> ln -fsr $TERMUX_PREFIX/games/davegnukem $TERMUX_PREFIX/bin/davegnukem
> ```

I don't think this will happen. If you build this package manually with `-i` flag, `$PREFIX` will be automatically deleted, and if with `-I` flag, this package will not be built again.

**Comment by IntinteDAO** [comment]
> > @licy183
> > One proposal. [1c55a30#diff-7a37231957f9339dcc9a65549560cc3897bc990fb9c097fe15215b15ceb3cd0bR19](https://github.com/termux-user-repository/tur/commit/1c55a30068c034e1f3992750823141f4907abd40#diff-7a37231957f9339dcc9a65549560cc3897bc990fb9c097fe15215b15ceb3cd0bR19)
> > I also suggest adding the -f parameter, because if you build the game 2 times, the link stays and during the second build, the build script detects an error and does not generate a new package.
> > so in short
> > ```
> > ln -fsr $TERMUX_PREFIX/games/davegnukem $TERMUX_PREFIX/bin/davegnukem
> > ```
> 
> I don't think this will happen. If you build this package manually with `-i` flag, `$PREFIX` will be automatically deleted, and if with `-I` flag, this package will not be built again.

Confirmed. I just used -f flag (in build-package), so it was a problem. But you can push it to mainline I think
